src/array_metric_types.py

```diff
@@ -14,8 +14,8 @@
 from db_action_response import DbActionResponse
 import score_table_models as stm
 from score_table_models import ArrayMetricType as amt
-from score_table_models import SatMeta as sm
-from sat_meta import SatMetaRequest
+from score_table_models import InstrumentMeta as im
+from instrument_meta import InstrumentMetaRequest
```

Comment:
I don't think there should be a name issue since they are imported as pretty different names. The table itself is imported as "im" and the handler request is "InstrumentMetaRequest". I find those to be fairly different and don't expect a developer to get them confused, especially as it follows the same pattern used in the rest of the repo.